RFS/src/main/java/org/opensearch/migrations/bulkload/workcoordination/SqlQueryBuilder.java

```diff
@@ -0,0 +1,157 @@
+package org.opensearch.migrations.bulkload.workcoordination;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+@SuppressWarnings("java:S1192") // Ignore duplicate literals in this file such as "UPDATE " or "INSERT INTO "
+class SqlQueryBuilder {
```

Comment:
Other than the .sql definition for the table, I wanted this to be the only place where we wrote sql strings.